src/KKS_MoreStoreItems/Features/EroDetectorFeat.cs

```diff
@@ -14,7 +14,8 @@ public class EroDetectorFeat : IFeature
     {
         private static ConfigEntry<bool> _notifyMast;
         private static ConfigEntry<bool> _notifyLesb;
-        private static string _infoTextPrefix = "エロ活動：{0}";
+        private static string _infoTextPrefixMast = "{0} オナニー：{1}"; //I am sorry if this is incorrect as i used google translate
+        private static string _infoTextPrefixLesb = "{0} レズビアン：{1}"; //Same here
```

Comment:
I think something like this would be more natural, but I'm worried it might be too long to read in time.
```
{0}は{1}でオナニーしている
{0}は{1}でセックスしている
```
Translations need to be added to the translation folder for these to have instant translations. Something like 
```
{0}は{1}でオナニーしている={0} is masturbating in {1}
{0}は{1}でセックスしている={0} is having sex in {1}
```
Not tested. Once you have a working set then they can be added to the translation repository in https://github.com/IllusionMods/KKS-Translation/blob/master/Translation/en/Text/Mods/Mod%20-%20General.txt